scripts/solve_network.py

```diff
@@ -118,10 +118,7 @@ def prepare_network(n, solve_opts, config):
             bus=n.buses.index,
             carrier="load",
             sign=1e-3,  # Adjust sign to measure p and p_nom in kW instead of MW

```

Comment:
To answer your question, this is used to specify the marginal cost below in terms of €/kWh as it is then divided 1e-3.

To be honest, it is best to avoid the use of this option. If we get rid of this line, below we can directly specify the value in €/MWh explicitly. The sign option may also be dropped in the future as it is not so used.
Would you mind doing that?

Reply:
Done :) I also adjusted p_nom accordingly.